common/prove/proof_keys.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/bnb-chain/zkbnb-crypto/circuit"
 	"github.com/bnb-chain/zkbnb-crypto/circuit/types"
+	types2 "github.com/bnb-chain/zkbnb/types"
```

Comment:
```suggestion
	zktypes "github.com/bnb-chain/zkbnb/types"
```
or even `ztypes`? `types2` is kinda ugly IMO.

Reply:
Yes. It is better. However, there are other places using the same notion. I will check whether change this in this pr or others.